src/pybind/mgr/dashboard_v2/controllers/perf_counters.py

```diff
@@ -0,0 +1,64 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+from ..tools import ApiController, AuthRequired, RESTController
+
+
+class PerfCounter(RESTController):
+    def __init__(self, service_type, mgr):
+        PerfCounter.mgr = mgr
+        self._service_type = service_type
+
+    @property
+    def service_type(self):
```

Comment:
why do you need to create a property for that instance attribute, is going to be used outside of this class?

Reply:
No, not really. I use the self._service_type instead now.